tests/scripts/basic-build-test.sh

```diff
@@ -35,6 +35,8 @@ if [ -d library -a -d include -a -d tests ]; then :; else
     exit 1
 fi
 
+MBEDTLS_ROOT_DIR="$PWD"
```

Comment:
This is a bit risky. It assumes that we are calling it from the actual root directory.

We could do something smart like asking git to tell us which is the root
`MBEDTLS_ROOT_DIR=$(git rev-parse --show-toplevel)`

But that won't work for tar-balls.

Reply:
Many of our test scripts either change to the root or must be called from the root. `basic-build-test.sh` in particular must be called from the root (see the check near the top of the file).

Reply:
> This is a bit risky. It assumes that we are calling it from the actual root directory.

We have checked that we are calling it from the root directory just above.